package/network/config/netifd/files/usr/libexec/network/packet-steering.sh

```diff
@@ -1,70 +1,75 @@
 #!/bin/sh
+
 NPROCS="$(grep -c "^processor.*:" /proc/cpuinfo)"
 [ "$NPROCS" -gt 1 ] || exit
 
-PROC_MASK="$(( (1 << $NPROCS) - 1 ))"
+PROC_MASK="$(( (1 << NPROCS) - 1 ))"
 
 find_irq_cpu() {
-	local dev="$1"
-	local match="$(grep -m 1 "$dev\$" /proc/interrupts)"
-	local cpu=0
-
-	[ -n "$match" ] && {
-		set -- $match
-		shift
-		for cur in $(seq 1 $NPROCS); do
-			[ "$1" -gt 0 ] && {
-				cpu=$(($cur - 1))
-				break
-			}
-			shift
-		done
-	}
+    dev="$1"
+    match="$(grep -m 1 "$dev\$" /proc/interrupts)"
+    cpu=0
 
-	echo "$cpu"
+    [ -n "$match" ] && {
+        set -- "$match"
+        shift
+        for cur in $(seq 1 "$NPROCS"); do
+            [ "$1" -gt 0 ] && {
+                cpu=$((cur - 1))
+                break
+            }
+            shift
+        done
+    }
+    echo "$cpu"
 }
 
 set_hex_val() {
-	local file="$1"
-	local val="$2"
-	val="$(printf %x "$val")"
-	[ -n "$DEBUG" ] && echo "$file = $val"
-	echo "$val" > "$file"
+    file="$1"
+    val="$2"
+    val="$(printf %x "$val")"
+    [ -n "$DEBUG" ] && echo "$file = $val"
+    echo "$val" > "$file"
 }
 
 packet_steering="$(uci get "network.@globals[0].packet_steering")"
 [ "$packet_steering" != 1 ] && exit 0
 
-exec 512>/var/lock/smp_tune.lock
-flock 512 || exit 1
+exec 9>/var/lock/smp_tune.lock
+flock 9 || exit 1
 
 [ -e "/usr/libexec/platform/packet-steering.sh" ] && {
-	/usr/libexec/platform/packet-steering.sh
-	exit 0
+    /usr/libexec/platform/packet-steering.sh
+    exit 0
 }
 
 for dev in /sys/class/net/*; do
-	[ -d "$dev" ] || continue
+    [ -d "$dev" ] || continue
+
+    # Ignore virtual interfaces
+    for file in "${dev}/lower_"*; do
+        if [ -e "$file" ]; then
+            continue
+        fi
+    done
 
-	# ignore virtual interfaces
-	[ -n "$(ls "${dev}/" | grep '^lower_')" ] && continue
-	[ -d "${dev}/device" ] || continue
+    [ -d "${dev}/device" ] || continue
 
-	device="$(readlink "${dev}/device")"
-	device="$(basename "$device")"
-	irq_cpu="$(find_irq_cpu "$device")"
-	irq_cpu_mask="$((1 << $irq_cpu))"
+    device="$(readlink "${dev}/device")"
+    device="$(basename "$device")"
+    irq_cpu="$(find_irq_cpu "$device")"
+    irq_cpu="$((1 << irq_cpu))"
```

Comment:
It appears this irq_cpu variable (and irq_cpu_mask before) is unused since d3868f15f876507db54afacdef22a7059011a54e. And therefore, `find_irq_cpu()` is also unused.